SimplenoteUITests/NoteEditor.swift

```diff
@@ -52,6 +52,17 @@ class NoteEditor {
         }
     }
 
+    class func enterTitle(enteredValue: String) {
+        let noteContentTextView = app.textViews.element
+
+        noteContentTextView.clearAndEnterText(text: enteredValue)
```

Comment:
This method does not add a new line after the first characters of note content (instead, you pass a string that already contains the `\n` in the end) so in its current state, it's no different from `NoteEditor.clearAndEnterText`.

In this case, adding this:

```suggestion
        noteContentTextView.clearAndEnterText(text: enteredValue + "\n")
```

Would add meaning to the method, and would also free you from the necessity of having the "\n" in the strings you define in test.

Reply:
Good point, totally didn't see this one. I'll update this to include the new line so that it doesn't have to be passed.